service-mesh/app-src/apps/demo/api.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+
+# API backend for the website.
+
+from flask import Flask, abort, jsonify, request
+import os
+import requests
+from signalfx_tracing import auto_instrument
+from signalfx_tracing.libraries import requests_config
+from . import utils
+
+
+tracer = utils.get_tracer()
+requests_config.propagate = True
+auto_instrument(tracer)
+app = Flask(__name__)
+
+
+@app.after_request
+def inject_trace_id(response):
+    scope = tracer.scope_manager.active
+    traceId = '{0:x}'.format(scope.span.context.trace_id)
+    response.headers['X-SF-TraceId'] = traceId
+    return response
```

Comment:
What is the purpose of this?

Reply:
Looks like maybe for debugging from the original source. Not having it didn't cause any issues, so it has been removed